tpbase/libc.go

```diff
@@ -169,12 +170,31 @@ func ExtractTSDInfoARM64(code []byte) (TSDInfo, error) {
 		resetReg = destReg
 		switch inst.Op {
 		case aa.MOV:
-			// Track register moves
-			srcReg, ok := ah.Xreg2num(inst.Args[1])
-			if !ok {
-				continue
+			var setImm bool
+			switch val := inst.Args[1].(type) {
+			case aa.Imm64:
```

Comment:
```suggestion
			case aa.Imm64, aa.Imm:
```
And remove the duplicate case block?